src/java/picard/vcf/filter/FilterVcf.java

```diff
@@ -87,14 +103,26 @@ public static void main(final String[] args) {
     }
 
     @Override
-    protected int doWork() {
+    public int doWork() {
```

Comment:
Did this have to change to be public?  I think the test should be moved into the same package and this be left as protected.  It's strange for a subclass to increase the visibility of a function.